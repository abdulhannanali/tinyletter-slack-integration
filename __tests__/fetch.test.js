const { TINYLETTER_BASE_URL, getArchiveUrl } = require('../tinyletterFetcher.js');

const { readArchive } = require('./helpers');
const { fetchArchive } = require('../tinyletterFetcher');

test('getArchiveUrl', () => {
    const url = getArchiveUrl('cassidoo');
    const url1 = getArchiveUrl('thedaggetts');

    expect(url).toMatchSnapshot();
    expect(url1).toMatchSnapshot();
});
