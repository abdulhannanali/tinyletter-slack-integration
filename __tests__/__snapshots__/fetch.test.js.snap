// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getArchiveUrl 1`] = `"https://tinyletter.com/cassidoo/archive"`;

exports[`getArchiveUrl 2`] = `"https://tinyletter.com/thedaggetts/archive"`;
